docs/source/ontologies_included.md

```diff
@@ -45,7 +35,55 @@ Start creating cuds objects:
 12
 ```
 
-## More ontologies
+## Dublin Core Metadata Initiative (DCMI)
+
+The `dcmitype` and `dcterms` RDFS vocabularies from the [Dublin Core 
+Metadata Initiative (DCMI)](https://www.dublincore.org/specifications/dublin-core/dcmi-terms/)
+can be quickly installed using
+
+```sh
+pico install dcmitype dcterms
+```
+
+Note that due to the fact that OSP-core [does not support RDFS properties](working_with_ontologies.html#limitations), 
+properties in these two vocabularies will be ignored by OSP-core. Only the 
+classes will be detected.
+
+## Data Catalog Vocabulary - Version 2 (DCAT2)
+
+To install the DCAT2 ontology, use
+
+```sh
+pico install dcat2 dcterms foaf prov 
+```
+
+Just installing `dcat2` will fail, as it depends on the `dcterms`, `foaf` 
+and `prov` vocabularies.
+
+## Friend of a Friend (FOAF)
+
+To install the [FOAF ontology](http://xmlns.com/foaf/spec/), use
+
+```sh
+pico install foaf
+```
+
+## The PROV Ontology (PROV-O)
+
+To install the [PROV-O ontology](https://www.w3.org/TR/prov-o/), use
+
+```sh
+pico install prov
+```
+
+## The City Ontology (city)
+
+It is a simple, example ontology included with OSP-core. It can be 
```

Comment:
Maybe add a reference to this: https://github.com/simphony/docs/blob/ae782d03ba78f7baaa816ad1f6c44f306e03aaa1/docs/source/ontology_intro.md?plain=1#L15

Reply:
The reference is on line 88 of this same file.

Reply:
Done